### DISCUSSION TITLE
why the uncompetitve inhibiton seem  to be wrong, when i change the Ki of uncompetitive inhibition, there is no change of AUC of viticim?

### DISCUSSION THREAD

**Discussion by dongji1982** [Open-Systems-Pharmacology/Forum#925] [Questions & Problems]
why the noncompetitve inhibiton mode seem  to be wrong? when i change the Ki of noncompetitive inhibition, there is no change of AUC of viticim?

**Comment by prvmalik** [comment]
First question would be: is the interaction applied when you Configure your simulation
